test/components/views/context_menus/MessageContextMenu-test.tsx

```diff
@@ -52,17 +237,19 @@ function createMessageContextMenu(eventContent: ExtensibleEvent) {
         },
     );
 
-    const mxEvent = new MatrixEvent(eventContent.serialize());
     mxEvent.setStatus(EventStatus.SENT);
 
     client.getUserId = jest.fn().mockReturnValue("@user:example.com");
     client.getRoom = jest.fn().mockReturnValue(room);
 
     return mount(
-        <MessageContextMenu
-            chevronFace={null}
-            mxEvent={mxEvent}
-            onFinished={jest.fn(() => {})}
-        />,
+        <RoomContext.Provider value={context as unknown as any}>
+            <MessageContextMenu
+                chevronFace={null}
+                mxEvent={mxEvent}
+                onFinished={jest.fn(() => {})}
```

Comment:
The `() => {}` bit shouldn't be needed, since that's the default implementation

Reply:
That doesn't seem to be the case - leaving it `undefined` results in an error

Reply:
It works for me locally :thinking:

```diff
-                onFinished={jest.fn(() => {})}
+                onFinished={jest.fn()}
```